### DISCUSSION TITLE
Allowed memory size of 134217728 bytes exhausted

### DISCUSSION THREAD

**Discussion by elray** [kimai/kimai#2996] [On-Premise: Installation & Updates] [answered]
Hi Kevin,

Trying to update from version 1.11 to 1.16.5. I'm running into Allowed memory size exhausted error when I try to clear cache.

 // Warming up the cache for the prod environment with debug false

PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 32768 bytes) in /var/www/html/kimai2/var/cache/prod/ContainerBmhEE44/getJmsSerializer_Cache_CacheWarmerService.php on line 1
19:24:26 CRITICAL  [php] Fatal Error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 32768 bytes) ["exception" => Symfony\Component\ErrorHandler\Error\OutOfMemoryError^ { …}]

In getJmsSerializer_Cache_CacheWarmerService.php line 1:

  Error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 32768 bytes)

Any ideas?

I can see the following in /var/log/prod.log ...

[prod.log](https://github.com/kevinpapst/kimai2/files/7663078/prod.log)

thanks for your help

Eliot

**Comment by kevinpapst** [comment, marked as answer]
Its right in the log: `failed to open stream: Permission denied` 
the documentation tells you how to do that

**Comment by elray** [comment]
Thanks Kevin. All working. 

I see a lot of 'php.INFO: User Deprecated: '  is it ok to ignore?

**Reply by kevinpapst** [reply]
Yes. You only take care of errors and I take care of the "Deprecations" :)
